test/XRP/xrp-client-integration.test.ts

```diff
@@ -71,36 +73,39 @@ describe('XRPClient Integration Tests', function (): void {
   > {
     this.timeout(timeoutMs)
 
-    const doesExist = await xrpWebClient.accountExists(recipientAddress)
-    assert.equal(doesExist, true)
+    const doesAccountExist = await xrpWebClient.accountExists(recipientAddress)
```

Comment:
Totally arbitrary renaming on my part because I like things to be a little more readable.